components/infobox/wikis/warcraft/infobox_league_custom.lua

```diff
@@ -98,16 +100,23 @@ function CustomLeague.run(frame)
 	return league:createInfobox(frame)
 end
 
+---@return WidgetInjector
 function CustomLeague:createWidgetInjector()
 	return CustomInjector()
 end
 
+---@param id string
+---@param widgets Widget[]
+---@return Widget[]
 function CustomInjector:parse(id, widgets)
 	if id == 'gamesettings' then
 		return {
 			Cell{name = 'Game', content = {GAMES[_args.game] and ('[[' .. GAMES[_args.game] .. ']]') or nil}},
-			Cell{name = 'Game version', content = {CustomLeague._displayGameVersion()}},
-			Cell{name = 'Server', content = {CustomLeague:_getServer()}}
+			Cell{name = 'Game version', content = {
+				CustomLeague._displayGameVersion(),
+				_args.patch2 and ('[[' .. _args.patch2 .. ']]') or nil
+			}},
+			Cell{name = 'Server', content = CustomLeague:_getServer()}
```

Comment:
```suggestion
			Cell{name = 'Server', content = _league:_getServer()}
```

Reply:
would also need to insert it into league